src/OpenTK.Mathematics/Vector/Vector2i.cs

```diff
@@ -96,6 +96,16 @@ public Vector2i(int x, int y)
             }
         }
 
+        /// <summary>
+        /// Gets the manhattan length of the vector.
+        /// </summary>
+        public int ManhattanLength => X + Y;
```

Comment:
This needs to be Abs(x) + Abs(y), otherwise it's not a length.

Reply:
Same for all of the other values.

Reply:
!!!!!
This is completely right!
Will fix.